### DISCUSSION TITLE
Switch to last active Tab with mouse-click

### DISCUSSION THREAD

**Discussion by KamelittaOida** [piroor/treestyletab#2875] [Ideas]
hello piroor

could you maybe add a feature that allows to alt+tab between two tabs by clicking the currently selected tab. ?
tab center reborn has such a feature.

![image](https://user-images.githubusercontent.com/29007971/116090916-a5b6cc80-a6a4-11eb-8160-f9a8844114c1.png)

**Comment by piroor** [comment]
Did you try [Tab flip for Tree Style Tab ](https://addons.mozilla.org/firefox/addon/tab-flip-for-tree-style-tab/) ?

**Reply by KamelittaOida** [reply]
thanks
